client/my-sites/email/titan-mail-quantity-selection/index.jsx

```diff
@@ -287,151 +224,140 @@ class TitanMailQuantitySelection extends React.Component {
+					external={ ! isEnabled( 'titan/iframe-control-panel' ) }
+					onClick={ this.handleCreateMailbox }
+				>
+					{ translate( 'Finish Setup' ) }
+				</NoticeAction>
+			</Notice>
+		);
+	}
+
+	renderNewMailboxesNotice() {
+		const { moment, selectedDomain, translate } = this.props;
+
+		if ( ! hasTitanMailWithUs( selectedDomain ) ) {
+			return null;
+		}
 
-		const pricingTitle = <h3>{ translate( 'Pricing' ) }</h3>;
-		// Handle cases where we're only dealing with the standard price
 		if ( this.doesAdditionalPriceMatchStandardPrice() ) {
-			return (
-				<React.Fragment>
-					{ pricingTitle }
-					<span>
-						{ translate( 'Each mailbox costs {{strong}}%(price)s/month{{/strong}}', {
-							args: {
-								price: titanMonthlyProduct.cost_display,
-							},
-							components: {
-								strong: <strong />,
-							},
-							comment:
-								'%(price)s is a formatted price for each mailbox, e.g. $3.50, €3.75, or PLN 3.75',
-						} ) }
-					</span>
-				</React.Fragment>
-			);
+			return;
 		}
 
 		const purchaseCost = getTitanMailboxPurchaseCost( selectedDomain );
 		const renewalCost = getTitanMailboxRenewalCost( selectedDomain );
 		const expiryDate = getTitanExpiryDate( selectedDomain );
 
-		const prorationMessage =
-			purchaseCost.amount < renewalCost.amount
-				? translate(
-						'Your initial purchase price is less than your standard monthly price of %(price)s per mailbox because ' +
-							"we're only charging you for the remainder of the current month.",
-						{
-							args: {
-								price: renewalCost.text,
-							},
-							comment:
-								'%(price)s is a formatted price for a monthly email mailbox subscription, e.g. $3.50, €3.75, or PLN 4.50',
-						}
-				  )
-				: translate(
-						'Your initial purchase price is more than your standard monthly price of %(price)s per mailbox because ' +
-							"we're charging you for the remainder of the current month plus one or more months of service.",
-						{
-							args: {
-								price: renewalCost.text,
-							},
-							comment:
-								'%(price)s is a formatted price for a monthly email mailbox subscription, e.g. $3.50, €3.75, or PLN 4.50',
-						}
-				  );
-
 		return (
-			<React.Fragment>
-				{ pricingTitle }
-				<span>
+			<Notice icon="info-outline" showDismiss={ false } status="is-success">
+				<>
 					{ translate(
-						'Each additional mailbox will cost {{strong}}%(price)s{{/strong}} for this purchase',
+						'You can purchase new mailboxes at the prorated price of {{strong}}%(proratedPrice)s{{/strong}} per mailbox.',
 						{
 							args: {
-								price: purchaseCost.text,
+								proratedPrice: purchaseCost.text,
 							},
 							components: {
 								strong: <strong />,
 							},
 							comment:
-								'%(price)s is a formatted price for each email mailbox, e.g. $3.50, €3.75, or PLN 4.50',
+								'%(proratedPrice)s is a formatted price for a monthly email mailbox subscription (e.g. $3.50, €3.75, or PLN 4.50)',
```

Comment:
I think it will be more future-proof to have the comment here not include `a monthly` as we will roll out annual plans at some stage, and we're unlikely to remember to fix/update this comment!
```suggestion
								'%(proratedPrice)s is a formatted price for an email mailbox subscription (e.g. $3.50, €3.75, or PLN 4.50)',
```

Reply:
Very good point!